includes/class-api.php

```diff
@@ -358,6 +361,13 @@ public function attach_featured_image( $post_id, $post_args ) {
 
 		$response = set_post_thumbnail( $post_id, $thumbnail_id );
 
+		// Remove filter that allowed download_url() to use an external host.
+		remove_filter( 'http_request_host_is_external', array( $this, 'allow_sync_external_host' ) );
+
+	}
+
+	public function allow_sync_external_host( $allow, $host, $url ) {
```

Comment:
@benjaminmueller you can just add a doc-block here to explain how it works. Also why does it just return true? Should it always return true by default?